firestore.rules

```diff
@@ -1,13 +1,53 @@
 rules_version = '2';
 service cloud.firestore {
   match /databases/{database}/documents {
+    function validAmount(docData){
```

Comment:
should I also add validation for `min/max` when the type is a certain type?

Reply:
I wouldn't worry about it. Something you could do here (and elsewhere) though is take in `amount` directly so you don't need to do `docData.amount`.